coordinator/src/node.rs

```diff
@@ -184,7 +181,8 @@ impl Node {
         )?;
 
         tracing::debug!(
-            "Settling position of {accept_settlement_amount} with {}",
+            ?position,
```

Comment:
Can we have the `peer_id` in her eas well please?

Reply:
yup.